### DISCUSSION TITLE
adjust the blank space in the page

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1882] [Ideas / Feature Requests]
how can we make the content width adjust to the page width,

![微信截图_20231026025936](https://github.com/zauberzeug/nicegui/assets/25096045/9141f5fc-f11a-43de-a486-dcb1ce89d07d)

not a problem,just feel a little uncomfortable about the blank area on the right

**Comment by v479038280** [comment]
I think the most tricky thing in nicegui is to adjust the classes about layout

**Comment by falkoschindler** [comment]
The layout might not be optimal. But it only affects these three demos on the main page.

Actually they use the same dimensions and breakpoints from the documentation page, which has a menu drawer, which is why there is some blank space on the main page.
